controller/locationcontroller.php

```diff
@@ -40,23 +39,22 @@ public function __construct($appName, IRequest $request,$locationManager,$userId
 	 * @CORS
 	 * @PublicPage
 	 */
-	public function update() {
-		$params = array('user' => $this -> userId);
-		$location['lat'] = $this->params('lat');
-		$location['lng'] = $this->params('lon');
-		if(((string)(float)$this->params('timestamp') === $this->params('timestamp'))) {
-			if(strtotime(date('d-m-Y H:i:s',$this->params('timestamp'))) === (int)$this->params('timestamp')) {
-				$location['timestamp'] = (int)$this->params('timestamp');
-			} elseif(strtotime(date('d-m-Y H:i:s',$this->params('timestamp')/1000)) === (int)floor($this->params('timestamp')/1000)) {
-				$location['timestamp'] = (int)floor($this->params('timestamp')/1000);
+	public function update($lat, $lon, $timestamp, $hdop, $altitude, $speed, $hash) {
```

Comment:
@LukasReschke I'd really like to hear about some of this cross site stuff from you.